packages/zowe-explorer/src/configuration/Profiles.ts

```diff
@@ -690,6 +627,7 @@ export class Profiles extends ProfilesCache {
         await this.openConfigFile(filePath);
     }
 
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
     public async profileValidationHelper(theProfile: imperative.IProfileLoaded, getStatus: (...args: unknown[]) => Promise<string>) {
```

Comment:
I saw that this line was added because the function has been in place for some time without a return type. Thoughts on updating the return type for this function to be explicit? It also eliminates the need for the ESLint comment.

```suggestion
    public async profileValidationHelper(theProfile: imperative.IProfileLoaded, getStatus: (...args: unknown[]) => Promise<string>): Promise<string> {
```